.github/docs/social-apps.md

```diff
@@ -11,22 +11,27 @@ This file contains the list of social app/website names which has public user pr
 -   [x] LinkedIn
 -   [ ] Reddit
 -   [ ] Spotify
+-   [x] Steam
 -   [x] Snapchat
 -   [x] Telegram
 -   [x] Tiktok
 -   [x] Tinder
+-   [x] Tumblr
 -   [x] Twitter
+-   [x] Pinterest
 -   [ ] WhatsApp
 -   [ ] X (twitter, but new logo & url)
 -   [x] YouTube
 
 ### Miscellaneous
 
 -   [x] Airchat
+-   [x] Ancestry
 -   [x] Bio.link
 -   [ ] Codechef
 -   [x] Codeforces
 -   [x] Dev.to
+-   [x] Exercism
```

Comment:
Just add the name of the app/website with `[ ]`, no need to check the checkbox, the checkbox is for some other purpose.